### PR TITLE
Enable the `rubocop-rake` and `rubocop-rspec` gems

### PR DESCRIPTION
**Pull request by smortex**
Rubocop is not configured to detect and help fixing all issues it can spot.  Enable this code to improve further the code of the gem.

Also include:
* #283 
* #284

### CODE REVIEW

File: tools/riemann-aws/lib/riemann/tools/aws/status.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ def tick
                     elsif opts[:event_warning] && (Date.today >= before - opts[:event_warning])
                       { state: 'warning' }
                     else
-                      { state: 'warning' }
+                      {}
```
Comment (smortex) [inline tools/riemann-aws/lib/riemann/tools/aws/status.rb:64]: I guess this is a bug?  Or maybe I miss something?

File: .rubocop.yml [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 AllCops:
+  NewCops: enable
   Exclude:
     - lib/riemann/tools/*_parser.tab.rb
     - vendor/bundle/**/*
+require:
+  - rubocop-rake
+  - rubocop-rspec
+Gemspec/RequireMFA:
+  Enabled: false
```
Comment (smortex) [inline .rubocop.yml:11]: I would prefer to require the usage of MFA, but according to rubygems.org "(some) owners are not using multi-factor authentication (MFA)."

**Comment by smortex** [comment]
This PR (and #283, #284 included in this PR) are a lot of small changes.  In order to make reviewing WIP work more easily (e.g. #282 where I hit these issues), I would like this PR to be reviewed and merged.  Thanks!

File: .rubocop.yml (hunk 2 above)
Comment (jamtur01) [reply .rubocop.yml:11]: It's @aphyr - when you get a chance, could you enable MFA on RubyGems if that works? Thanks!

**Review by jamtur01 — APPROVED** [review]
LGTM from what I can see. I don't take issues with any of the cleanup.

File: .rubocop.yml (hunk 2 above)
Comment (aphyr) [reply .rubocop.yml:11]: Done!

**Pull request merged**